jamjar/jamjar/videos/views.py

```diff
@@ -3,11 +3,32 @@
 from rest_framework.authtoken.models import Token
 from jamjar.base.views import BaseView
 
+from rest_framework.parsers import FormParser, MultiPartParser
+
 from jamjar.videos.models import Video
 from jamjar.videos.serializers import VideoSerializer
 
-class VideoList(BaseView):
+import uuid
+
+from django.http import HttpResponse
+from wsgiref.util import FileWrapper
+
+
+class VideoStream(BaseView):
+    def get(self, request, id):
+         # Attempt to get the video
+        try:
+            self.video = Video.objects.get(id=id)
+        except:
+            return self.error_response('Video does not exist or you do not have access to this video.', 404)
 
+        file = FileWrapper(open(self.video.src, 'rb'))
```

Comment:
i didn't try to make it too generic because we'll ultimately be using S3 to store videos. Agree that a constant is the way to go, but this will all change pretty soon anyway